docs/sources/features/datasources/cloudmonitoring.md

```diff
@@ -324,3 +324,14 @@ datasources:
     jsonData:
       authenticationType: gce
 ```
+
+## Deep linking from Grafana panels to the Metrics Explorer in Gooogle Cloud Console
+
+Only available in Grafana v7.1+.
+
+{{< docs-imagebox img="/img/docs/v71/cloudmonitoring_deep_linking.png" max-width="500px" class="docs-image--right" caption="Google Cloud Monitoring deep linking" >}}
+
+> **Note:** This feature is only available for Metric queries.
+
+Click on a time series in the panel to see a context menu with a link to View in Metrics explorer in Google Cloud Console. Clicking that link opens the Metrics explorer in the Google Cloud Console and runs the query from the Grafana panel there.
+The link navigates the user first to the Google Account Chooser and after successfully selecting an account, the user is redirected to the metrics explorer. The provided link is valid for any account, but it only displays the query if your account has access to the GCP project specified in the query.
```

Comment:
Inconsistent capitalization of "metrics explorer" here. If it is a proper noun (named in the UI), then it should be capitalized to match the UI. If it is not, then lowercase throughout. Either way, try to be consistent.